internal/auth/authenticator.go

```diff
@@ -180,8 +183,12 @@ func (p *Authenticator) newMux() http.Handler {
 	serviceMux.HandleFunc("/validate", p.withMethods(p.validateClientID(p.validateClientSecret(p.ValidateToken)), "GET"))
 	serviceMux.HandleFunc("/redeem", p.withMethods(p.validateClientID(p.validateClientSecret(p.Redeem)), "POST"))
 	serviceMux.HandleFunc("/refresh", p.withMethods(p.validateClientID(p.validateClientSecret(p.Refresh)), "POST"))
-	// serve static files but prevent directory listings, see static_files.go
-	serviceMux.Handle("/static/", http.StripPrefix("/static/", http.FileServer(noDirectoryFS{http.Dir("./static")})))
+	fsHandler, err := loadFSHandler()
+	if err != nil {
+		logger.Fatal(err)
+		os.Exit(1)
```

Comment:
Actually, I meant to check whether `.Fatal` calls `os.Exit` or not before committing. Looks like it does, so this line isn't needed.

Reply:
That said this is one of those "this should never happen" errors.